packages/bot-web-ui/src/components/dashboard/dashboard-component/cards.tsx

```diff
@@ -1,62 +1,171 @@
+                method: () => setActiveTab(2),
+                disable: '',
+            },
+        ];
+        SetIconArray(IconArray);
+    }, [has_file_loaded]);
 
     return (
-        <div className='dc-tabs__dashboard-cards-wrapper' id='db-dashboard-cards'>
-            {actions.map(({ icon, content, method }, index) => (
-                <div key={index} className='dc-tabs__dashboard-card'>
-                    <Icon
-                        className='dc-tabs__dashboard-card__image'
-                        width='8rem'
-                        height='8rem'
-                        icon={icon}
-                        id={icon}
-                        onClick={method}
-                    />
-                    <Text size='xxs' line_height='s' color='prominent'>
-                        {localize(content)}
-                    </Text>
-                </div>
-            ))}
+        <div className='tab__dashboard__table'>
+            <div
+                className='dc-tabs__content_group_tiles tab__dashboard__table__tiles'
+                id='dc-tabs__content_group_tiles'
+            >
+                {icon_array.map((icons, index) => {
+                    const { icon, content, method, disable } = icons;
+                    return (
+                        <div
+                            key={index}
+                            className={`dc-tabs__content_group_tiles_block ${disable} tab__dashboard__table__block`}
+                        >
+                            <Icon
+                                className={'dc-tabs__content_group_tiles_images tab__dashboard__table__images'}
+                                width='8rem'
+                                height='8rem'
+                                icon={icon}
+                                id={icon}
+                                onClick={method}
+                            />
+
+                            <span className='dc-tabs__content_group_tiles_content'>{localize(content)}</span>
+                        </div>
+                    );
+                })}
+                <input
+                    type='file'
+                    ref={file_input_ref}
+                    accept='.xml'
+                    style={{ display: 'none' }}
+                    onChange={e => {
+                        //setLoadedLocalFile(null);
+                        clear_preview_ref.current?.click();
+                        onConfirmSave();
+                        setIsFileSupported(handleFileChange(e, false));
+                        loadFileFromLocal();
+                        setFileLoaded(true);
+                    }}
+                />
+                <Dialog
+                    title={dialog_options.title}
+                    is_visible={is_dialog_open}
+                    onCancel={closeResetDialog}
+                    is_mobile_full_width
+                    className={'dc-dialog__wrapper--fixed dc-dialog__wrapper--googledrive'}
```

Comment:
I couldn't find the css for dc-dialog__wrapper--fixed

Reply:
done ✅

Reply:
done ✅